app/views/wikies/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>Editing Wiky</h1>
-
 <%= render 'form' %>
+<%= render 'collaborations/form', collaborations: @collaboration, wiky: @wiky  %><br>
```

Comment:
I changed it. @collaborations and @wiky come from the wikies/controller
edit action

On Wed, Feb 11, 2015 at 1:02 PM, Eliot Sykes <notifications@github.com>
wrote:

> In app/views/wikies/edit.html.erb
> <https://github.com/LuisMenjivar/wiki/pull/48#discussion_r24517016>:
>
> >  <%= render 'form' %>
> > +<%= render 'collaborations/form', collaborations: @collaboration, wiky: @wiky  %><br>
>
> This is a little confusing, where plural and singular names are mixed: collaborations:
> @collaboration
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/wiki/pull/48/files#r24517016>.
>